packages/cli/generators/app/templates/src/controllers/ping.controller.ts.ejs

```diff
@@ -1,14 +1,41 @@
 import {Request, RestBindings, get} from '@loopback/rest';
 import {inject} from '@loopback/context';
 
+const PING_RESPONSE = {
```

Comment:
Please add an explicit type `SchemaObject` to let the compiler verify correctness of the schema here, where we are defining `PING_RESPONSE`, instead of deferring the check until `PING_RESPONSE` is assigned to `responses` in `@get` below.

```ts
const PING_RESPONSE: SchemaObject = {
  // ...
};
```